madhumitha/images.html

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>image</title>
+</head>
+<body>
+    <div>
+        <img id="image_id"></img>
+    </div>
+    <div>
+        <button id="button">Show Image</button>
+    </div>
+    <script>
+        document.addEventListener("DOMContentLoaded", function () {
+            let x = document.getElementById("button");
+            x.addEventListener("click", function () {
+                var ImgArray = [
+                    "http://farm4.staticflickr.com/3691/11268502654_f28f05966c_m.jpg",
+

```

Comment:
ImgArray is getting declared again each time on click listener. Can you have it globally?

Reply:
Yes, I have changed the code and pushed it